cli/cli.go

```diff
@@ -148,18 +148,19 @@ OPTIONS:
 		Usage:       "manage and start the Mender client.",
 		Version:     ShowVersion(),
 	}
-	app.Commands = []cli.Command{
+	app.Commands = []*cli.Command{
 		{
 			Name:  "bootstrap",
 			Usage: "Perform bootstrap and exit.",
 			Flags: []cli.Flag{
-				cli.BoolFlag{
-					Name:        "forcebootstrap, F",
+				&cli.BoolFlag{
+					Name:        "forcebootstrap",
+					Aliases:     []string{"F"},
 					Usage:       "Force bootstrap.",
 					Destination: &runOptions.bootstrapForce},
 			},
 			Action: func(ctx *cli.Context) error {
-				if len(ctx.Args()) > 0 {
+				if ctx.Args().Len() > 0 {
```

Comment:
All these checks can be moved down to a single check in `commonCLIHandler` function to reduce the cyclometric complexity of this function.

Reply:
Done. I'll push it in the upcoming rework.

Reply:
Please have a close look at [this commit](https://github.com/mendersoftware/mender/pull/499/commits/b5f95c79e3188d3c93ef5278d72adc87e0c4a210).